Gemfile

```diff
@@ -26,6 +26,10 @@ gem 'simple_form'
 gem 'devise'
 gem 'cancancan'
 
+gem 'bootstrap', '~> 4.5.0'
+gem 'jquery-rails'
+
```

Comment:
@sansuaza code consistency (extra spaces) is relevant when we talk about code quality.

Reply:
solved!